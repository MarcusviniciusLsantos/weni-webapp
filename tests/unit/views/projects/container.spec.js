import { shallowMount, createLocalVue } from '@vue/test-utils';
import Container from '@/views/projects/container.vue';
import i18n from '@/utils/plugins/i18n';

const localVue = createLocalVue();

describe('AppGrid.vue', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallowMount(Container, {
      localVue,
      i18n,
      mocks: {
        $t: () => 'some specific text',
      },
    });
  });

  it('should be rendered properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
