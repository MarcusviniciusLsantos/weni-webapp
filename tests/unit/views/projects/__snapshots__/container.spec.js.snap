// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppGrid.vue should be rendered properly 1`] = `
<div class="screen">
  <div class="container">
    <div class="unnnic-grid-lg">
      <div class="content"></div>
    </div>
  </div>
</div>
`;
